src/QMCDrivers/RMC/RMCUpdatePbyP.cpp

```diff
@@ -552,6 +408,11 @@ namespace qmcplusplus
 	W.reptile->flip ();
 	// return;
       }
+      //Collectables should be evaluated on center bead.  Here we go.
+      Walker_t& centerbead = W.reptile->getCenter();
+      W.loadWalker(centerbead,true);
+      W.update(false);  //Called to recompute S(k) and distance tables.  Recompute S(k)?  False.
```

Comment:
@rcclay false means skipSK=false. update your comment and remove false(default).